lib/server/routes/health.js

```diff
@@ -23,7 +27,7 @@ HealthRouter.prototype.health = function(req, res) {
 
 HealthRouter.prototype._definitions = function() {
   return [
-    ['GET', '/health', this.health]
+    ['GET', '/health', this.getLimiter(limiter(1000)), this.health]
```

Comment:
We could boost the total to something fairly high, like 10,000 per minute. I guess it might make sense to remove it. I'll leave it up to your call.